src/test/java/dataStructures/bTree/BTreeTest.java

```diff
@@ -0,0 +1,30 @@
+package dataStructures.bTree;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BTreeTest {
+    @Test
+    public void test_BTree() {
```

Comment:
could you also add 1 more test case - just to make sure the structure of the tree is correct. Check against the pre-order and level-order display of the tree. Will probably need a get pre and level order method in ur main code.